Multisig/UI/App/UIKit/UI Library/ReusableViews/AddressInfoView.swift

```diff
@@ -8,18 +8,29 @@
 
 import UIKit
 
-class AddressWithTitleView: UINibView {
+struct AddressInfo {
+    let address: Address
+    let label: String?
+}
+
+class AddressInfoView: UINibView {
     @IBOutlet private weak var identiconView: UIImageView!
     @IBOutlet private weak var textLabel: UILabel!
     @IBOutlet private weak var detailLabel: UILabel!
-    @IBOutlet private weak var button: UIButton!
+    @IBOutlet private weak var copyButton: UIButton!
```

Comment:
this is named "copyButton" but the action is named "onAddressInfoButton" - which is the correct one?

Reply:
renamed action into `copyAddress`